code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm

```diff
@@ -752,11 +752,11 @@
 		spitting = TRUE
 		if(xeno.ammo.pre_spit_warn)
 			playsound(xeno.loc,"alien_drool", 55, 1)
-		to_chat(xeno, SPAN_WARNING("You begin to prepare a large spit!"))
+		to_chat(xeno, SPAN_WARNING("We begin to prepare a large spit!"))
```

Comment:
redundant with the visible_message below i guess

Reply:
one is for you one is for everyone so i'll just keep it

Reply:
The second part of the visible message is just for you. Won't this just be doubled up in the chat?